.travis.yml

```diff
@@ -21,22 +21,28 @@ matrix:
       env: SYMFONY_VERSION=4.0.*
     - php: 7.2
       env: SYMFONY_VERSION=3.4.*
+    - php: 7.2
+      env:
+        - SYMFONY_VERSION=4.0.*
+        - FLYSYSTEM_VERSION=1.0.*
+        - SIMPLE_PHPUNIT_PARAMETERS=" --stop-on-skipped"
```

Comment:
i think this is not doing what you think it does. it stops execution when a skipped test is encountered, but not treat it as failure. see https://travis-ci.org/liip/LiipImagineBundle/jobs/326390446 

it looks like a [--fail-on-skipped](https://github.com/sebastianbergmann/phpunit/issues/2840) is considered as unimportant by sebastian, so we could either maybe do a complicated thing with a test listener to detect skipped tests, or use an environment variable that we define on the line that runs phpunit that we look at in those tests and if its present fail instead of skip.